app/views/experiments/new.html.haml

```diff
@@ -1,7 +1,7 @@
 - es_url = sample_service_url('experiment_supervisors')
 - supervisor_ids = get_supervisor_ids(es_url)
 - wss_algorithms_descriptions = workers_scaling_algorithms_description
-- plgrid_default_enabled = !InfrastructureFacadeFactory.get_facade_for(:qsub).get_subinfrastructures(current_user.id).blank?
+- plgrid_default_enabled = !InfrastructureFacadeFactory.get_facade_for(:qsub).get_infrastructure_configurations(current_user.id).blank?
```

Comment:
@kliput Wiem że chciałeś tutaj zmienić get_infrastructure_configurations na enabled_for_user? jednak nie jest to jednoznaczne:
```
2.1.5 :004 > InfrastructureFacadeFactory.get_facade_for(:qsub).enabled_for_user?(ScalarmUser.first.id)
 => true 
2.1.5 :005 > InfrastructureFacadeFactory.get_facade_for(:qsub).get_infrastructure_configurations(ScalarmUser.first.id)
 => [] 
2.1.5 :006 > 
```
Zakładam że ma to związek z usuwaniem credentiali użytkownika po określonym czasie?